specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2022-09-10/service.json

```diff
@@ -4460,7 +4460,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/replicationProtectionContainers/{protectionContainerName}/replicationProtectedItems/{replicationProtectedItemName}/updateMobilityService": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{resourceName}/replicationFabrics/{fabricName}/replicationProtectionContainers/{protectionContainerName}/replicationProtectedItems/{replicatedProtectedItemName}/updateMobilityService": {
```

Comment:
>subscriptions [](http://example.com/codeflow?start=6&length=13)

please get the breaking change approved after which ARM can sign off

Reply:
@rkmanda From what I know, this is not a breakinge change from API's perspective.

Ping @leni-msft

Reply:
@JeffreyRichter could you check if renaming *path parameter* is breaking change from API's perspective?

Reply:
This is potentially breaking in C# SDKs but is so unlikely that we consider it non-breaking.

Reply:
@rkmanda Any update on this PR? I saw there is another similar PR got merged month ago: https://github.com/Azure/azure-rest-api-specs/pull/20588.